ci/debian.9.5.amd64.sh

```diff
@@ -0,0 +1,76 @@
+set -x
+echo "CI: Ubuntu 16.04 x86_64"
```

Comment:
Shouldn't this say "debian something"?

Reply:
it probably should :)

I will submit a corrected version, that was a copy from the other file.

Reply:
@pallas1 fixed, please check again.